_database-integrations/mysql-amazon-rds.md

```diff
@@ -0,0 +1,137 @@
+tags: [database_integrations]
+permalink: /integrations/databases/amazon-rds-mysql
+summary: "Connect and replicate data from your Amazon RDS MysQL database using Stitch's MySQL integration."
+
+# -------------------------- #
+#     Integration Details    #
+# -------------------------- #
+
+name: "mysql-rds"
+display_name: "MySQL RDS"
+author: "Stitch"
+author-url: "https://www.stitchdata.com"
+
+# -------------------------- #
+#       Stitch Details       #
+# -------------------------- #
+
+status: "Released"
+certified: true
+
+frequency: "30 minutes"
+tier: "Free"
+port: 3306
+db-type: "mysql"
+icon: /images/integrations/icons/amazon-rds.svg
+
+versions: "n/a"
+ssh: true
+ssl: true
+sync-views: true
+whitelist:
+  tables: true
+  columns: true
+
+setup-name: "MySQL"
+
+# -------------------------- #
+#      Setup Requirements    #
+# -------------------------- #
+
+requirements-list:
+  - item: |
+      **Permissions in Amazon Web Services (AWS) that allow you to**:
+
+        - Create/manage Security Groups, which is required to whitelist Stitch's IP addresses.
+        - View database details, which is required for retrieving the database's connection details.
+  - item: "**The `CREATE USER` or `INSERT` privilege (for the `mysql` database).** The [`CREATE USER` privilege](https://dev.mysql.com/doc/refman/8.0/en/create-user.html) is required to create a database user for Stitch."
+  - item: "**The `GRANT OPTION` privilege in {{ integration.display_name }}.** The [`GRANT OPTION` privilege](https://dev.mysql.com/doc/refman/8.0/en/privileges-provided.html#priv_grant-option) is required to grant the necessary privileges to the Stitch database user."
+
+# -------------------------- #
+#     Setup Instructions     #
+# -------------------------- #
+
+setup-steps:
+  - title: "whitelist stitch ips"
+
+  - title: "retrieve public key"
+
+  - title: "create linux user"
+
+  - title: "Configure database server settings"
+    anchor: "server-settings"
+    content: |
+      {% include integrations/databases/setup/binlog/configure-server-settings-intro.html %}
+    substeps:
+      - title: "Configure the database parameter group"
+        anchor: "configure-database-parameter-group"
+        content: |
+          {% include integrations/databases/setup/binlog/amazon-rds.html %}
+
+      - title: "Define the backup retention period"
+        anchor: "define-backup-retention-period"
+        content: |
+          The backup retention period setting defines the number of days for which automated backups are retained. This ensures that data can still be replicated even if issues with Stitch arise.
+
+          1. Navigate back to the **Instances** page by using the menu on the left side of the page.
+          2. Select the instance you're connecting to Stitch.
+          3. Click **Instance actions > Modify**.
+          4. Scroll down to the **Backup** section.
+          5. Set **Backup retention period** to `1 day`:
```

Comment:
This doesn't necessarily need to be 1 day specifically. I believe it just needs to be something non-zero.